resources/README.md

```diff
@@ -0,0 +1,10 @@
+# Resources
+
+A repository for various courses, guides, blog posts that we find useful for our
+development processes.
```

Comment:
I don't really think we should restrict this to development. Otherwise, good initiative.